Jenkinsfile

```diff
@@ -18,7 +18,7 @@ pipeline {
     }
     stage('Archive artifacts') {
       steps {
-        archiveArtifacts artifacts: 'releng/plugins/org.polarsys.capella.studio.releng.product/target/*.txt, releng/plugins/org.polarsys.capella.studio.releng.updatesite/target/repository/**, releng/plugins/org.polarsys.capella.studio.releng.updatesite/target/*.txt'
+        archiveArtifacts artifacts: 'releng/plugins/org.polarsys.capella.studio.releng.product/target/products/*.zip, releng/plugins/org.polarsys.capella.studio.releng.product/target/*.txt, releng/plugins/org.polarsys.capella.studio.releng.updatesite/target/repository/**, releng/plugins/org.polarsys.capella.studio.releng.updatesite/target/*.txt'
```

Comment:
You must not archive capella studio zip products on jenkins, 
those large artefacts must be (and are already) uploaded to https://download.eclipse.org/capella/capellastudio/products/nightly/
You can upload them to nightly/<branch-PR-name> if you want to backup them for each pull request 
(https://download.eclipse.org/capella/core/products/nightly/) 

Thanks

Reply:
So I'll move artefacts to a better place.